# Titanic competition with TensorFlow Decision Forests

Credit to this file belongs to @gusthema and @achoum666 on Kaggle.  I reproduced their work to learn from the pros!

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition. 

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

2024-04-25 20:52:52.776536: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-04-25 20:52:52.776670: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-04-25 20:52:52.929334: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered


Found TF-DF 1.8.1


# Load dataset

In [2]:
train_df = pd.read_csv("/kaggle/input/titanic/train.csv")
serving_df = pd.read_csv("/kaggle/input/titanic/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)


PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [5]:
def tokenize_names(features, labels=None):
    """Divide the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [6]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-04-25 20:53:06.8178 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:06.8188 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:06.8189 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:53:14.3237 UTC kernel.cc:1233] Loading model from path /tmp/tmpc7_5f5t6/model/ with prefix 8247ad935d1d48ea
[INFO 24-04-25 20:53:14.3313 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:53:14.3318 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:53:14.3318 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.8260869383811951 Loss:0.8608942627906799


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")


[WARNING 24-04-25 20:53:22.7880 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:22.7881 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:22.7881 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:53:23.7645 UTC kernel.cc:1233] Loading model from path /tmp/tmp15cn9lnk/model/ with prefix 95f73e0601eb4424
[INFO 24-04-25 20:53:23.7751 UTC decision_forest.cc:660] Model loaded with 40 root(s), 2106 node(s), and 10 input feature(s).
[INFO 24-04-25 20:53:23.7753 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.782608687877655 Loss:1.0586705207824707


In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.585997 ################
    2.           "Age"  0.364636 #######
    3.          "Fare"  0.266191 ###
    4.          "Name"  0.207054 #
    5.        "Pclass"  0.179191 
    6. "Ticket_number"  0.178806 
    7.      "Embarked"  0.177803 
    8.   "Ticket_item"  0.177009 
    9.         "Parch"  0.175276 
   10.         "SibSp"  0.171694 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 34.000000 ######

# Make predictions

In [9]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="/kaggle/working/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head /kaggle/working/submission.csv

Submission exported to /kaggle/working/submission.csv


/opt/conda/lib/python3.10/pty.py:89: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.
  pid, fd = os.forkpty()


PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).

In [10]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")


Use /tmp/tmpwc_j_kzd as temporary training directory


[WARNING 24-04-25 20:53:25.5880 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:25.5880 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:53:25.5880 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:32.7169 UTC kernel.cc:1233] Loading model from path /tmp/tmpwc_j_kzd/model/ with prefix 4f1641fbb57b41e6
[INFO 24-04-25 20:55:32.7386 UTC decision_forest.cc:660] Model loaded with 30 root(s), 2236 node(s), and 12 input feature(s).
[INFO 24-04-25 20:55:32.7387 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-04-25 20:55:32.7387 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.8630136847496033 Loss:0.6749962568283081


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [11]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)


i:0


[WARNING 24-04-25 20:55:47.9923 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:47.9924 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:47.9924 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:48.6266 UTC kernel.cc:1233] Loading model from path /tmp/tmp25ddl84g/model/ with prefix a26b90ccc7be4924
[INFO 24-04-25 20:55:48.6312 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:55:48.6316 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:55:48.6316 UTC kernel.cc:1061] Use fast generic engine


i:1


[WARNING 24-04-25 20:55:49.1294 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:49.1294 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:49.1294 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:50.0706 UTC kernel.cc:1233] Loading model from path /tmp/tmpow78o0xm/model/ with prefix 4517d12f13ee4388
[INFO 24-04-25 20:55:50.0912 UTC kernel.cc:1061] Use fast generic engine


i:2


[WARNING 24-04-25 20:55:50.5951 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:50.5952 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:50.5952 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:51.1641 UTC kernel.cc:1233] Loading model from path /tmp/tmpmt2f0svn/model/ with prefix 5152281b8edc4ba8
[INFO 24-04-25 20:55:51.1690 UTC kernel.cc:1061] Use fast generic engine


i:3


[WARNING 24-04-25 20:55:51.6709 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:51.6710 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:51.6710 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:52.8018 UTC kernel.cc:1233] Loading model from path /tmp/tmporfd4q0w/model/ with prefix a403098e038340f5
[INFO 24-04-25 20:55:52.8292 UTC kernel.cc:1061] Use fast generic engine


i:4


[WARNING 24-04-25 20:55:53.3376 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:53.3376 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:53.3376 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:53.9259 UTC kernel.cc:1233] Loading model from path /tmp/tmpmdphlu53/model/ with prefix f572de8b025141da
[INFO 24-04-25 20:55:53.9318 UTC kernel.cc:1061] Use fast generic engine


i:5


[WARNING 24-04-25 20:55:54.4261 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:54.4262 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:54.4262 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:54.9425 UTC kernel.cc:1233] Loading model from path /tmp/tmp36ns5gov/model/ with prefix e929d7c668cd4c39
[INFO 24-04-25 20:55:54.9457 UTC kernel.cc:1061] Use fast generic engine


i:6


[WARNING 24-04-25 20:55:55.5155 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:55.5155 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:55.5155 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:56.1619 UTC kernel.cc:1233] Loading model from path /tmp/tmp6ecxt0mt/model/ with prefix c3c9b25d33d84497
[INFO 24-04-25 20:55:56.1699 UTC kernel.cc:1061] Use fast generic engine


i:7


[WARNING 24-04-25 20:55:56.6705 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:56.6706 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:56.6706 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:57.6583 UTC kernel.cc:1233] Loading model from path /tmp/tmpk016p5_a/model/ with prefix ed0896256d5a4a78
[INFO 24-04-25 20:55:57.6805 UTC kernel.cc:1061] Use fast generic engine


i:8


[WARNING 24-04-25 20:55:58.4089 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:58.4089 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:55:58.4090 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:55:59.1110 UTC kernel.cc:1233] Loading model from path /tmp/tmpf_t4y4_9/model/ with prefix a76a61fecc9944dd
[INFO 24-04-25 20:55:59.1217 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:55:59.1217 UTC kernel.cc:1061] Use fast generic engine


i:9


[WARNING 24-04-25 20:56:00.1479 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:00.1479 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:00.1479 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:00.9854 UTC kernel.cc:1233] Loading model from path /tmp/tmpy9hq3a5f/model/ with prefix 6294423f47a64c31
[INFO 24-04-25 20:56:01.0017 UTC kernel.cc:1061] Use fast generic engine


i:10


[WARNING 24-04-25 20:56:01.5098 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:01.5098 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:01.5098 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:02.1013 UTC kernel.cc:1233] Loading model from path /tmp/tmppsuoitpj/model/ with prefix 2b9b915af7d54956
[INFO 24-04-25 20:56:02.1071 UTC kernel.cc:1061] Use fast generic engine


i:11


[WARNING 24-04-25 20:56:02.6247 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:02.6247 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:02.6247 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:03.4513 UTC kernel.cc:1233] Loading model from path /tmp/tmpag7qhk11/model/ with prefix 6e41527dad20488b
[INFO 24-04-25 20:56:03.4666 UTC kernel.cc:1061] Use fast generic engine


i:12


[WARNING 24-04-25 20:56:03.9692 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:03.9693 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:03.9693 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:04.5748 UTC kernel.cc:1233] Loading model from path /tmp/tmp67vilsj2/model/ with prefix c2e7871484f24ee1
[INFO 24-04-25 20:56:04.5811 UTC kernel.cc:1061] Use fast generic engine


i:13


[WARNING 24-04-25 20:56:05.0850 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:05.0851 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:05.0852 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:05.8429 UTC kernel.cc:1233] Loading model from path /tmp/tmpr4fmfznc/model/ with prefix 14f21ca6d8814d9e
[INFO 24-04-25 20:56:05.8545 UTC kernel.cc:1061] Use fast generic engine


i:14


[WARNING 24-04-25 20:56:06.3504 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:06.3504 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:06.3505 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:07.4461 UTC kernel.cc:1233] Loading model from path /tmp/tmpwgrgi1dz/model/ with prefix 98fd4faa73aa4150
[INFO 24-04-25 20:56:07.4535 UTC kernel.cc:1061] Use fast generic engine


i:15


[WARNING 24-04-25 20:56:07.9855 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:07.9855 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:07.9855 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:08.6343 UTC kernel.cc:1233] Loading model from path /tmp/tmp8xevrns7/model/ with prefix 36960c5e41994df2
[INFO 24-04-25 20:56:08.6431 UTC kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-04-25 20:56:09.6877 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:09.6877 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:09.6877 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:10.4725 UTC kernel.cc:1233] Loading model from path /tmp/tmp2bc19ro6/model/ with prefix b411c9dade9041cd
[INFO 24-04-25 20:56:10.4864 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:56:10.4865 UTC kernel.cc:1061] Use fast generic engine


i:17


[WARNING 24-04-25 20:56:11.4253 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:11.4253 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:11.4253 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:12.2433 UTC kernel.cc:1233] Loading model from path /tmp/tmpxe3i45kj/model/ with prefix a30e22054fb24d8b
[INFO 24-04-25 20:56:12.2577 UTC kernel.cc:1061] Use fast generic engine


i:18


[WARNING 24-04-25 20:56:12.7659 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:12.7660 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:12.7660 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:13.5222 UTC kernel.cc:1233] Loading model from path /tmp/tmp6r83rvbj/model/ with prefix c1b7b63b0e3a47d5
[INFO 24-04-25 20:56:13.5347 UTC kernel.cc:1061] Use fast generic engine


i:19


[WARNING 24-04-25 20:56:14.0355 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:14.0355 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:14.0355 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:14.9528 UTC kernel.cc:1233] Loading model from path /tmp/tmp_yhig4qm/model/ with prefix 39caa96dd0754bc6
[INFO 24-04-25 20:56:14.9721 UTC kernel.cc:1061] Use fast generic engine


i:20


[WARNING 24-04-25 20:56:15.7890 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:15.7890 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:15.7891 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:16.6187 UTC kernel.cc:1233] Loading model from path /tmp/tmp8_dmbzrr/model/ with prefix 936d8cdac949421c
[INFO 24-04-25 20:56:16.6347 UTC kernel.cc:1061] Use fast generic engine


i:21


[WARNING 24-04-25 20:56:17.1424 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:17.1425 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:17.1425 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:17.7196 UTC kernel.cc:1233] Loading model from path /tmp/tmpd6c6guwu/model/ with prefix fd8f631290084ecb
[INFO 24-04-25 20:56:17.7251 UTC kernel.cc:1061] Use fast generic engine


i:22


[WARNING 24-04-25 20:56:18.2313 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:18.2313 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:18.2313 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:18.8214 UTC kernel.cc:1233] Loading model from path /tmp/tmpa3c9dbt2/model/ with prefix c58323f9e2ee4925
[INFO 24-04-25 20:56:18.8272 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:56:18.8276 UTC kernel.cc:1061] Use fast generic engine


i:23


[WARNING 24-04-25 20:56:19.9931 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:19.9931 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:19.9931 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:20.6996 UTC kernel.cc:1233] Loading model from path /tmp/tmp5oqqlbtm/model/ with prefix e66e81ede8414394
[INFO 24-04-25 20:56:20.7091 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:56:20.7092 UTC kernel.cc:1061] Use fast generic engine


i:24


[WARNING 24-04-25 20:56:21.2204 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:21.2205 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:21.2205 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:21.8189 UTC kernel.cc:1233] Loading model from path /tmp/tmpqope429l/model/ with prefix 72639ae142254f59
[INFO 24-04-25 20:56:21.8250 UTC kernel.cc:1061] Use fast generic engine


i:25


[WARNING 24-04-25 20:56:22.3433 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:22.3434 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:22.3434 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:23.1073 UTC kernel.cc:1233] Loading model from path /tmp/tmp_f6e5wuv/model/ with prefix e2ae4f6d26314f84
[INFO 24-04-25 20:56:23.1193 UTC kernel.cc:1061] Use fast generic engine


i:26


[WARNING 24-04-25 20:56:23.6254 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:23.6255 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:23.6255 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:24.3446 UTC kernel.cc:1233] Loading model from path /tmp/tmpt28b2vzt/model/ with prefix 91b45d021d484964
[INFO 24-04-25 20:56:24.3553 UTC kernel.cc:1061] Use fast generic engine


i:27


[WARNING 24-04-25 20:56:24.8603 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:24.8604 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:24.8604 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:25.4691 UTC kernel.cc:1233] Loading model from path /tmp/tmpur1eyua2/model/ with prefix d10e4941ca964885
[INFO 24-04-25 20:56:25.4759 UTC kernel.cc:1061] Use fast generic engine


i:28


[WARNING 24-04-25 20:56:26.5963 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:26.5963 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:26.5964 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:27.1772 UTC kernel.cc:1233] Loading model from path /tmp/tmpreoepa8r/model/ with prefix 329b71f1c92c4596
[INFO 24-04-25 20:56:27.1827 UTC kernel.cc:1061] Use fast generic engine


i:29


[WARNING 24-04-25 20:56:27.6856 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:27.6856 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:27.6856 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:28.4789 UTC kernel.cc:1233] Loading model from path /tmp/tmpz5do3gk5/model/ with prefix 340ebb4016de4bfe
[INFO 24-04-25 20:56:28.4925 UTC kernel.cc:1061] Use fast generic engine


i:30


[WARNING 24-04-25 20:56:29.0063 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:29.0065 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:29.0065 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:30.1503 UTC kernel.cc:1233] Loading model from path /tmp/tmpl20mld3d/model/ with prefix 2d786bb494734bcc
[INFO 24-04-25 20:56:30.1793 UTC kernel.cc:1061] Use fast generic engine


i:31


[WARNING 24-04-25 20:56:30.6891 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:30.6892 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:30.6892 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:31.3767 UTC kernel.cc:1233] Loading model from path /tmp/tmpwsm20hjk/model/ with prefix 30d1a5fd2c04416d
[INFO 24-04-25 20:56:31.3868 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:56:31.3868 UTC kernel.cc:1061] Use fast generic engine


i:32


[WARNING 24-04-25 20:56:31.8874 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:31.8875 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:31.8875 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:32.4967 UTC kernel.cc:1233] Loading model from path /tmp/tmp20s9uswb/model/ with prefix 8f26f4340af44345
[INFO 24-04-25 20:56:32.5029 UTC kernel.cc:1061] Use fast generic engine


i:33


[WARNING 24-04-25 20:56:33.0114 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:33.0114 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:33.0115 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:33.8079 UTC kernel.cc:1233] Loading model from path /tmp/tmpat67mh5e/model/ with prefix cb84d27e53dd43fc
[INFO 24-04-25 20:56:33.8217 UTC kernel.cc:1061] Use fast generic engine


i:34


[WARNING 24-04-25 20:56:34.3219 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:34.3220 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:34.3220 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:34.9793 UTC kernel.cc:1233] Loading model from path /tmp/tmpbddbr59d/model/ with prefix 15c0d108ffbb40e6
[INFO 24-04-25 20:56:34.9883 UTC kernel.cc:1061] Use fast generic engine


i:35


[WARNING 24-04-25 20:56:36.0573 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:36.0573 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:36.0574 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:36.6944 UTC kernel.cc:1233] Loading model from path /tmp/tmpyoinsb9o/model/ with prefix 7c4a61cd7d7144c6
[INFO 24-04-25 20:56:36.7026 UTC kernel.cc:1061] Use fast generic engine


i:36


[WARNING 24-04-25 20:56:37.7897 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:37.7897 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:37.7897 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:38.6414 UTC kernel.cc:1233] Loading model from path /tmp/tmph8onnut0/model/ with prefix bfe89798d99743db
[INFO 24-04-25 20:56:38.6571 UTC kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-04-25 20:56:39.1610 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:39.1611 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:39.1611 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:39.8280 UTC kernel.cc:1233] Loading model from path /tmp/tmp_th1tm8o/model/ with prefix 9245e448bb3c4afd
[INFO 24-04-25 20:56:39.8374 UTC kernel.cc:1061] Use fast generic engine


i:38


[WARNING 24-04-25 20:56:40.8944 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:40.8944 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:40.8945 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:41.6952 UTC kernel.cc:1233] Loading model from path /tmp/tmp1hxbolwx/model/ with prefix bac8b253574940bb
[INFO 24-04-25 20:56:41.7091 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:56:41.7091 UTC kernel.cc:1061] Use fast generic engine


i:39


[WARNING 24-04-25 20:56:42.2104 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:42.2104 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:42.2105 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:42.9971 UTC kernel.cc:1233] Loading model from path /tmp/tmpn67pfeu_/model/ with prefix a4856b91bbc34d57
[INFO 24-04-25 20:56:43.0110 UTC kernel.cc:1061] Use fast generic engine


i:40


[WARNING 24-04-25 20:56:45.2436 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:45.2436 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:45.2436 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:45.8801 UTC kernel.cc:1233] Loading model from path /tmp/tmpe9tnp4t1/model/ with prefix 55c0701fd1b1431d
[INFO 24-04-25 20:56:45.8855 UTC kernel.cc:1061] Use fast generic engine


i:41


[WARNING 24-04-25 20:56:46.4206 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:46.4207 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:46.4207 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:47.2997 UTC kernel.cc:1233] Loading model from path /tmp/tmpi5xhocnq/model/ with prefix aeeb2e5f605b4ff8
[INFO 24-04-25 20:56:47.3187 UTC kernel.cc:1061] Use fast generic engine


i:42


[WARNING 24-04-25 20:56:47.8511 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:47.8512 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:47.8512 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:48.5661 UTC kernel.cc:1233] Loading model from path /tmp/tmpn7660sl1/model/ with prefix ae17584f4d6e4b08
[INFO 24-04-25 20:56:48.5767 UTC kernel.cc:1061] Use fast generic engine


i:43


[WARNING 24-04-25 20:56:49.5660 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:49.5660 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:49.5660 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:50.5000 UTC kernel.cc:1233] Loading model from path /tmp/tmp47vnevtg/model/ with prefix f280c8391c184cc4
[INFO 24-04-25 20:56:50.5183 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:56:50.5197 UTC kernel.cc:1061] Use fast generic engine


i:44


[WARNING 24-04-25 20:56:51.0557 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:51.0557 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:51.0558 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:51.8261 UTC kernel.cc:1233] Loading model from path /tmp/tmp3od9qlvz/model/ with prefix 6d29b0a7fb734752
[INFO 24-04-25 20:56:51.8367 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:56:51.8367 UTC kernel.cc:1061] Use fast generic engine


i:45


[WARNING 24-04-25 20:56:52.3568 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:52.3569 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:52.3569 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:52.9336 UTC kernel.cc:1233] Loading model from path /tmp/tmpzgepfzss/model/ with prefix 14c29e353a8d4acd
[INFO 24-04-25 20:56:52.9374 UTC kernel.cc:1061] Use fast generic engine


i:46


[WARNING 24-04-25 20:56:53.4583 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:53.4583 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:53.4584 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:54.2850 UTC kernel.cc:1233] Loading model from path /tmp/tmpnkriblrp/model/ with prefix 1dd5ff8c079e4d2a
[INFO 24-04-25 20:56:54.3003 UTC kernel.cc:1061] Use fast generic engine


i:47


[WARNING 24-04-25 20:56:54.8007 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:54.8007 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:54.8008 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:55.6097 UTC kernel.cc:1233] Loading model from path /tmp/tmpfs1s59j8/model/ with prefix 8645bfe2b814404e
[INFO 24-04-25 20:56:55.6241 UTC kernel.cc:1061] Use fast generic engine


i:48


[WARNING 24-04-25 20:56:56.5474 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:56.5474 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:56.5474 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:57.1100 UTC kernel.cc:1233] Loading model from path /tmp/tmp37j825mh/model/ with prefix f4280cee09f841a8
[INFO 24-04-25 20:56:57.1148 UTC kernel.cc:1061] Use fast generic engine


i:49


[WARNING 24-04-25 20:56:57.5995 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:57.5996 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:57.5996 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:58.2243 UTC kernel.cc:1233] Loading model from path /tmp/tmpx3zv4qse/model/ with prefix f7d6423872ba4001
[INFO 24-04-25 20:56:58.2316 UTC kernel.cc:1061] Use fast generic engine


i:50


[WARNING 24-04-25 20:56:58.7305 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:58.7305 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:58.7305 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:56:59.4812 UTC kernel.cc:1233] Loading model from path /tmp/tmpwewbn_s2/model/ with prefix 1db94bda7b4940d2
[INFO 24-04-25 20:56:59.4941 UTC kernel.cc:1061] Use fast generic engine


i:51


[WARNING 24-04-25 20:56:59.9941 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:59.9941 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:56:59.9941 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:00.8694 UTC kernel.cc:1233] Loading model from path /tmp/tmpuskaakzy/model/ with prefix ae7a16adb37d4e8f
[INFO 24-04-25 20:57:00.8859 UTC kernel.cc:1061] Use fast generic engine


i:52


[WARNING 24-04-25 20:57:01.3996 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:01.3996 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:01.3996 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:02.0581 UTC kernel.cc:1233] Loading model from path /tmp/tmp6827x3u2/model/ with prefix b23e0aad31db40ac
[INFO 24-04-25 20:57:02.0663 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:02.0663 UTC kernel.cc:1061] Use fast generic engine


i:53


[WARNING 24-04-25 20:57:03.1537 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:03.1537 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:03.1538 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:03.8089 UTC kernel.cc:1233] Loading model from path /tmp/tmplvgrmfuf/model/ with prefix f12dab8f47c040ad
[INFO 24-04-25 20:57:03.8169 UTC kernel.cc:1061] Use fast generic engine


i:54


[WARNING 24-04-25 20:57:04.3317 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:04.3320 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:04.3321 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:04.8771 UTC kernel.cc:1233] Loading model from path /tmp/tmpnsfs1j_e/model/ with prefix 6f1a1b682a434759
[INFO 24-04-25 20:57:04.8806 UTC kernel.cc:1061] Use fast generic engine


i:55


[WARNING 24-04-25 20:57:05.3780 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:05.3780 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:05.3780 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:06.2561 UTC kernel.cc:1233] Loading model from path /tmp/tmpjkiyb2e7/model/ with prefix 86a05d9c6a664c55
[INFO 24-04-25 20:57:06.2714 UTC kernel.cc:1061] Use fast generic engine


i:56


[WARNING 24-04-25 20:57:06.7786 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:06.7787 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:06.7787 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:07.5182 UTC kernel.cc:1233] Loading model from path /tmp/tmp2kvu5txx/model/ with prefix 80176255a77e4bdd
[INFO 24-04-25 20:57:07.5294 UTC kernel.cc:1061] Use fast generic engine


i:57


[WARNING 24-04-25 20:57:08.0346 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:08.0346 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:08.0346 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:08.6128 UTC kernel.cc:1233] Loading model from path /tmp/tmpijqy9wio/model/ with prefix 380823a16ffc4a70
[INFO 24-04-25 20:57:08.6173 UTC kernel.cc:1061] Use fast generic engine


i:58


[WARNING 24-04-25 20:57:09.1135 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:09.1136 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:09.1136 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:09.7220 UTC kernel.cc:1233] Loading model from path /tmp/tmpts9jogkb/model/ with prefix 0ef236c6f65346b3
[INFO 24-04-25 20:57:09.7289 UTC kernel.cc:1061] Use fast generic engine


i:59


[WARNING 24-04-25 20:57:10.2269 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:10.2269 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:10.2269 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:10.9209 UTC kernel.cc:1233] Loading model from path /tmp/tmp6nxr1bkx/model/ with prefix d94bafa498be4510
[INFO 24-04-25 20:57:10.9305 UTC kernel.cc:1061] Use fast generic engine


i:60


[WARNING 24-04-25 20:57:11.9739 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:11.9739 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:11.9739 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:12.6779 UTC kernel.cc:1233] Loading model from path /tmp/tmp8c23nq0y/model/ with prefix 0c7706138c024205
[INFO 24-04-25 20:57:12.6883 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:12.6884 UTC kernel.cc:1061] Use fast generic engine


i:61


[WARNING 24-04-25 20:57:13.1859 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:13.1859 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:13.1860 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:13.7714 UTC kernel.cc:1233] Loading model from path /tmp/tmpwizwsim7/model/ with prefix 5db46df9750f4cad
[INFO 24-04-25 20:57:13.7767 UTC kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-04-25 20:57:14.2940 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:14.2941 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:14.2943 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:15.0695 UTC kernel.cc:1233] Loading model from path /tmp/tmpzmia6l8j/model/ with prefix 845a68d7da8c4d75
[INFO 24-04-25 20:57:15.0827 UTC kernel.cc:1061] Use fast generic engine


i:63


[WARNING 24-04-25 20:57:15.5813 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:15.5814 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:15.5814 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:16.2894 UTC kernel.cc:1233] Loading model from path /tmp/tmpkzu49jdh/model/ with prefix 826e65b49ac24438
[INFO 24-04-25 20:57:16.2992 UTC kernel.cc:1061] Use fast generic engine


i:64


[WARNING 24-04-25 20:57:16.8024 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:16.8025 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:16.8025 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:17.4567 UTC kernel.cc:1233] Loading model from path /tmp/tmp0uhfo9qa/model/ with prefix a1585b2374744723
[INFO 24-04-25 20:57:17.4648 UTC kernel.cc:1061] Use fast generic engine


i:65


[WARNING 24-04-25 20:57:17.9617 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:17.9617 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:17.9617 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:18.5328 UTC kernel.cc:1233] Loading model from path /tmp/tmpcl34tvgy/model/ with prefix 849e8b73731e4b73
[INFO 24-04-25 20:57:18.5378 UTC kernel.cc:1061] Use fast generic engine


i:66


[WARNING 24-04-25 20:57:19.0283 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:19.0283 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:19.0284 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:20.3789 UTC kernel.cc:1233] Loading model from path /tmp/tmpmf9r2e0r/model/ with prefix 5c3f575d381e4383
[INFO 24-04-25 20:57:20.3877 UTC kernel.cc:1061] Use fast generic engine


i:67


[WARNING 24-04-25 20:57:21.4665 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:21.4665 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:21.4665 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:22.4643 UTC kernel.cc:1233] Loading model from path /tmp/tmpdn1534oo/model/ with prefix e1b486ab529e4128
[INFO 24-04-25 20:57:22.4864 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:57:22.4884 UTC kernel.cc:1061] Use fast generic engine


i:68


[WARNING 24-04-25 20:57:23.0611 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:23.0613 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:23.0615 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:23.8922 UTC kernel.cc:1233] Loading model from path /tmp/tmpgc84e0yh/model/ with prefix 1ecd1646da2c41ef
[INFO 24-04-25 20:57:23.9069 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:23.9070 UTC kernel.cc:1061] Use fast generic engine


i:69


[WARNING 24-04-25 20:57:24.8675 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:24.8675 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:24.8675 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:25.4907 UTC kernel.cc:1233] Loading model from path /tmp/tmpkqao8vwn/model/ with prefix e4d423ff55714e39
[INFO 24-04-25 20:57:25.4979 UTC kernel.cc:1061] Use fast generic engine


i:70


[WARNING 24-04-25 20:57:26.0345 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:26.0345 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:26.0346 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:26.7071 UTC kernel.cc:1233] Loading model from path /tmp/tmp2jtjvjv5/model/ with prefix 9ec4bd82d288468d
[INFO 24-04-25 20:57:26.7168 UTC kernel.cc:1061] Use fast generic engine


i:71


[WARNING 24-04-25 20:57:27.2259 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:27.2260 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:27.2260 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:27.8700 UTC kernel.cc:1233] Loading model from path /tmp/tmp5q9oles3/model/ with prefix 439a6941aae74311
[INFO 24-04-25 20:57:27.8789 UTC kernel.cc:1061] Use fast generic engine


i:72


[WARNING 24-04-25 20:57:28.3876 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:28.3876 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:28.3876 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:29.2579 UTC kernel.cc:1233] Loading model from path /tmp/tmp0634msev/model/ with prefix d980747c104144db
[INFO 24-04-25 20:57:29.2748 UTC kernel.cc:1061] Use fast generic engine


i:73


[WARNING 24-04-25 20:57:30.1418 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:30.1419 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:30.1419 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:30.7723 UTC kernel.cc:1233] Loading model from path /tmp/tmpcrnyykwj/model/ with prefix 9fdd1f92f19f498a
[INFO 24-04-25 20:57:30.7801 UTC kernel.cc:1061] Use fast generic engine


i:74


[WARNING 24-04-25 20:57:31.8960 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:31.8960 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:31.8961 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:32.6658 UTC kernel.cc:1233] Loading model from path /tmp/tmpdu4jkr4m/model/ with prefix 32dd15eaf4e64f9d
[INFO 24-04-25 20:57:32.6785 UTC kernel.cc:1061] Use fast generic engine


i:75


[WARNING 24-04-25 20:57:33.6503 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:33.6503 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:33.6503 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:34.3089 UTC kernel.cc:1233] Loading model from path /tmp/tmpe848ciyh/model/ with prefix 443f1778f1ab4652
[INFO 24-04-25 20:57:34.3174 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:34.3174 UTC kernel.cc:1061] Use fast generic engine


i:76


[WARNING 24-04-25 20:57:34.8102 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:34.8103 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:34.8103 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:35.3618 UTC kernel.cc:1233] Loading model from path /tmp/tmp55yguqza/model/ with prefix 81ce32dc708e4216
[INFO 24-04-25 20:57:35.3662 UTC kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-04-25 20:57:35.8955 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:35.8956 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:35.8956 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:36.4678 UTC kernel.cc:1233] Loading model from path /tmp/tmph1etwvzy/model/ with prefix 0f7595f011684397
[INFO 24-04-25 20:57:36.4730 UTC kernel.cc:1061] Use fast generic engine


i:78


[WARNING 24-04-25 20:57:36.9962 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:36.9963 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:36.9964 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:37.6089 UTC kernel.cc:1233] Loading model from path /tmp/tmpci9jsyn0/model/ with prefix c7f98035e46341bc
[INFO 24-04-25 20:57:37.6160 UTC kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-04-25 20:57:38.1164 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:38.1165 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:38.1165 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:38.7419 UTC kernel.cc:1233] Loading model from path /tmp/tmpz2k1y1tt/model/ with prefix 5dfb88068ff549bc
[INFO 24-04-25 20:57:38.7494 UTC kernel.cc:1061] Use fast generic engine


i:80


[WARNING 24-04-25 20:57:39.2454 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:39.2455 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:39.2455 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:39.9435 UTC kernel.cc:1233] Loading model from path /tmp/tmpe4vanyun/model/ with prefix 855089ff854a492b
[INFO 24-04-25 20:57:39.9540 UTC kernel.cc:1061] Use fast generic engine


i:81


[WARNING 24-04-25 20:57:40.9841 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:40.9842 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:40.9842 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:41.7396 UTC kernel.cc:1233] Loading model from path /tmp/tmpdx8b06vx/model/ with prefix 51a0bfb512b94a7d
[INFO 24-04-25 20:57:41.7518 UTC kernel.cc:1061] Use fast generic engine


i:82


[WARNING 24-04-25 20:57:42.2613 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:42.2614 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:42.2614 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:42.9810 UTC kernel.cc:1233] Loading model from path /tmp/tmp_8m0d3_4/model/ with prefix efb9cf977d274adb
[INFO 24-04-25 20:57:42.9919 UTC kernel.cc:1061] Use fast generic engine


i:83


[WARNING 24-04-25 20:57:44.0160 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:44.0160 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:44.0160 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:44.7050 UTC kernel.cc:1233] Loading model from path /tmp/tmp6fkhiqtw/model/ with prefix b9e78616a4f14fb5
[INFO 24-04-25 20:57:44.7145 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:44.7146 UTC kernel.cc:1061] Use fast generic engine


i:84


[WARNING 24-04-25 20:57:45.2091 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:45.2091 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:45.2091 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:46.0883 UTC kernel.cc:1233] Loading model from path /tmp/tmpk5ttc5k3/model/ with prefix 6043d55542bd435b
[INFO 24-04-25 20:57:46.1049 UTC kernel.cc:1061] Use fast generic engine


i:85


[WARNING 24-04-25 20:57:46.6105 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:46.6106 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:46.6107 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:47.2355 UTC kernel.cc:1233] Loading model from path /tmp/tmpa20sdlnb/model/ with prefix 603b503977534ea2
[INFO 24-04-25 20:57:47.2428 UTC kernel.cc:1061] Use fast generic engine


i:86


[WARNING 24-04-25 20:57:47.7498 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:47.7499 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:47.7499 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:48.6225 UTC kernel.cc:1233] Loading model from path /tmp/tmpr5o2429t/model/ with prefix d21a9990c5254a9b
[INFO 24-04-25 20:57:48.6397 UTC kernel.cc:1061] Use fast generic engine


i:87


[WARNING 24-04-25 20:57:49.1410 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:49.1410 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:49.1410 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:50.0715 UTC kernel.cc:1233] Loading model from path /tmp/tmp0z5v11bs/model/ with prefix 0ad85c37a7fd4bf6
[INFO 24-04-25 20:57:50.0905 UTC kernel.cc:1061] Use fast generic engine


i:88


[WARNING 24-04-25 20:57:50.8977 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:50.8978 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:50.8978 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:51.6475 UTC kernel.cc:1233] Loading model from path /tmp/tmp70u9bxj0/model/ with prefix 9c87a795917c4385
[INFO 24-04-25 20:57:51.6596 UTC kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-04-25 20:57:52.1603 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:52.1603 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:52.1603 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:52.7559 UTC kernel.cc:1233] Loading model from path /tmp/tmprkdf1cfa/model/ with prefix 5532712be79d40dc
[INFO 24-04-25 20:57:52.7608 UTC quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-04-25 20:57:52.7612 UTC kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-04-25 20:57:53.2889 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:53.2891 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:53.2891 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:54.0162 UTC kernel.cc:1233] Loading model from path /tmp/tmp6grxf6ac/model/ with prefix 9dff7deaa50a4ce6
[INFO 24-04-25 20:57:54.0268 UTC kernel.cc:1061] Use fast generic engine


i:91


[WARNING 24-04-25 20:57:55.0665 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:55.0665 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:55.0666 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:55.7699 UTC kernel.cc:1233] Loading model from path /tmp/tmpwqb9nsib/model/ with prefix e8567818f1124628
[INFO 24-04-25 20:57:55.7772 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:57:55.7772 UTC kernel.cc:1061] Use fast generic engine


i:92


[WARNING 24-04-25 20:57:56.8312 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:56.8312 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:56.8312 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:57.7751 UTC kernel.cc:1233] Loading model from path /tmp/tmpte2web9x/model/ with prefix dc9c7be08f7b4f69
[INFO 24-04-25 20:57:57.7947 UTC kernel.cc:1061] Use fast generic engine


i:93


[WARNING 24-04-25 20:57:58.3021 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:58.3022 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:57:58.3022 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:57:59.0257 UTC kernel.cc:1233] Loading model from path /tmp/tmpell55qr8/model/ with prefix c74092fd604f48f1
[INFO 24-04-25 20:57:59.0362 UTC kernel.cc:1061] Use fast generic engine


i:94


[WARNING 24-04-25 20:58:00.3229 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:00.3230 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:00.3230 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:01.0466 UTC kernel.cc:1233] Loading model from path /tmp/tmpq3r80zil/model/ with prefix 380fa61ae62e4ad4
[INFO 24-04-25 20:58:01.0560 UTC kernel.cc:1061] Use fast generic engine


i:95


[WARNING 24-04-25 20:58:01.6179 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:01.6179 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:01.6179 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:02.3678 UTC kernel.cc:1233] Loading model from path /tmp/tmpcb5esua1/model/ with prefix 03463790a05c453a
[INFO 24-04-25 20:58:02.3781 UTC kernel.cc:1061] Use fast generic engine


i:96


[WARNING 24-04-25 20:58:02.9377 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:02.9377 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:02.9378 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:03.7348 UTC kernel.cc:1233] Loading model from path /tmp/tmpoixwnv3a/model/ with prefix ad122998553c4c7a
[INFO 24-04-25 20:58:03.7472 UTC kernel.cc:1061] Use fast generic engine


i:97


[WARNING 24-04-25 20:58:04.2829 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:04.2829 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:04.2829 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:04.9006 UTC kernel.cc:1233] Loading model from path /tmp/tmp6s9czbmd/model/ with prefix f29470d6dbd84911
[INFO 24-04-25 20:58:04.9065 UTC kernel.cc:1061] Use fast generic engine


i:98


[WARNING 24-04-25 20:58:05.4424 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:05.4424 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:05.4424 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:06.1651 UTC kernel.cc:1233] Loading model from path /tmp/tmp1caziz8k/model/ with prefix 199b57aceb9148e6
[INFO 24-04-25 20:58:06.1739 UTC abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-04-25 20:58:06.1739 UTC kernel.cc:1061] Use fast generic engine


i:99


[WARNING 24-04-25 20:58:06.7281 UTC gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:06.7281 UTC gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-04-25 20:58:06.7281 UTC gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-04-25 20:58:07.5817 UTC kernel.cc:1233] Loading model from path /tmp/tmptos7ur59/model/ with prefix 3a8fb825880548ab
[INFO 24-04-25 20:58:07.5973 UTC kernel.cc:1061] Use fast generic engine


Submission exported to /kaggle/working/submission.csv


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests) 